src/components/form/form.pw.tsx

```diff
@@ -0,0 +1,387 @@
+        mount,
+        page,
+      }) => {
+        await mount(<FormWithFullWidthButtons fullWidthButtons={bool} />);
+
+        const submitButton = getComponent(page, "button").filter({
+          hasText: "submit",
+        });
+        await assertCssValueIsApproximately(submitButton, "width", buttonWidth);
+      });
+    }
+  );
+});
+
+test.describe("Accessibility tests for Form component", () => {
+  test(`should pass tests for DefaultWithStickyFooter example`, async ({
+    mount,
+    page,
+  }) => {
+    await mount(<DefaultWithStickyFooter />);
+
+    await checkAccessibility(page);
+  });
+
+  test(`should pass tests for FormAlignmentExample`, async ({
+    mount,
+    page,
+  }) => {
+    await mount(<FormAlignmentExample />);
+
+    await checkAccessibility(page);
+  });
+
+  test(`should pass tests for InDialog example`, async ({ mount, page }) => {
+    await mount(<InDialog />);
+
+    const dialogButton = dataComponentButtonByText(page, "Open Preview");
+    await dialogButton.click();
+    await checkAccessibility(page);
+  });
+
+  test(`should pass tests for InDialogFullScreen example`, async ({
+    mount,
+    page,
+  }) => {
+    await mount(<InDialogFullScreen />);
+
+    const dialogButton = dataComponentButtonByText(page, "Open Preview");
+    await dialogButton.click();
+    await checkAccessibility(page);
+  });
+
+  test(`should pass tests for InDialogFullScreenWithStickyFooter example`, async ({
+    mount,
+    page,
+  }) => {
+    await mount(<InDialogFullScreenWithStickyFooter />);
+
+    const dialogButton = dataComponentButtonByText(page, "Open Preview");
+    await dialogButton.click();
+    await checkAccessibility(page);
+  });
+
+  test(`should pass tests for InDialogWithStickyFooter example`, async ({
+    mount,
+    page,
+  }) => {
+    await mount(<InDialogWithStickyFooter />);
+
+    const dialogButton = dataComponentButtonByText(page, "Open Preview");
+    await dialogButton.click();
+    await checkAccessibility(page);
+  });
+
+  test(`should pass tests for OverrideFieldSpacing example`, async ({
+    mount,
+    page,
+  }) => {
+    await mount(<OverrideFieldSpacing />);
+
+    await checkAccessibility(page);
+  });
+
+  test(`should pass tests for WithAdditionalButtons example`, async ({
+    mount,
+    page,
+  }) => {
+    await mount(<WithAdditionalButtons />);
+
+    await checkAccessibility(page);
+  });
+
+  test(`should pass tests for WithBothErrorsAndWarningsSummary example`, async ({
+    mount,
+    page,
+  }) => {
+    await mount(<WithBothErrorsAndWarningsSummary />);
+
+    await checkAccessibility(page);
+  });
+
+  test(`should pass tests for WithButtonsAlignedToTheLeft example`, async ({
+    mount,
+    page,
+  }) => {
+    await mount(<WithButtonsAlignedToTheLeft />);
+
+    await checkAccessibility(page);
+  });
+
+  test(`should pass tests for WithCustomFooterPadding example`, async ({
```

Comment:
This should click on the `Open Preview` button first as currently it's just testing the default frame with the `Open Preview` button and not the Form with the custom footer padding.